images/noobaa/Makefile

```diff
@@ -0,0 +1,36 @@
+# Copyright 2016 The Rook Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+include ../image.mk
+
+# ====================================================================================
+# Image Build Options
+
+NOOBAA_IMAGE = $(BUILD_REGISTRY)/noobaa-$(GOARCH)
+
+TEMP := $(shell mktemp -d)
+
+# ====================================================================================
+# Build Rook
+
+do.build:
+	@echo === docker build $(NOOBAA_IMAGE)
+	@cp Dockerfile $(TEMP)
+	@cp $(OUTPUT_DIR)/bin/linux_$(GOARCH)/rook $(TEMP)/rook-noobaa-operator
```

Comment:
I think of the rook binary as a collection of tools, so here in the build process for the noobaa operator image I thought it made sense that the process name will be descriptive as well. Maybe its because I'm used to looking at the linux level. In any way the process name inside the container is not really important.

**TLDR: Changed. Resolving.**